_src/deploy-to-github.js

```diff
@@ -0,0 +1,14 @@
+/** Fixes deployment URLs when on a preview URL */
+document.addEventListener('DOMContentLoaded', () => {
+  const host = window.location.hostname;
+  if (host !== 'docs.chain.link') {
+    // Rewrite Remix URLs with current hostname
+    // eg https://remix.ethereum.org/#version=soljson-v0.6.7+commit.b8d736ae.js&optimize=false&evmVersion=null&url=https://docs.chain.link/samples/VRF/VRFD20.sol
+
+    for (let item of document.links) {
+      if (item.href.startsWith('https://remix.ethereum.org')) {
+        item.setAttribute('href', item.href.replace('docs.chain.link', host));
```

Comment:
The way we currently have `yarn serve` configured run Browsersync, it uses http by default instead of https. Forcing `https://` into the browser URL just returns `ERR_SSL_PROTOCOL_ERROR`.

Can you modify this to detect and apply `http://` or `https://` depending on what the host is using? Either that or we need to set up Browsersync to just redirect.

Reply:
I guess we're not specifically worried about local testing, which likely doesn't have ports 443 or 80 open externally anyway. Just good to remember that some Remix links won't work when using `yarn serve`.